en/views/helpers/form.rst

```diff
@@ -516,6 +516,17 @@ Options for Select, Checkbox and Radio Inputs
 
     The value key for date and datetime inputs may also be a UNIX
     timestamp, or a DateTime object.
+    
+  
+
+  For select input where you turn on the `multiple` attribute as true, 
+  you can use an array of the values you want to render as default selected::
+    
+    echo $this->Form->select('rooms', [
+        'multiple' => true,
+        'value' => [1, 3] // options with values 1 and 3 will be selected as default
```

Comment:
Shoudln't you use the 'default' key? Using value usually breaks forms on invalidation.